IDinsight/aaq-core
# Gaurdrails tests

Reviewer: @suzinyou 

Estimate: 1 hr

----

# Adding test for rails

## Ticket

Fixes: [AAQ-224](https://idinsight.atlassian.net/browse/AAQ-224)

## Description, Motivation and Context

### Goal

In the previous PR, I added rails. These add tests we can run when we change models to make sure 
the rails still run.

### Changes
- new tests
- edits to ensure it is not run automatically or in CI/CD since it calls openai and therefore costs money

## How Has This Been Tested?

* Make sure you have exported your OPENAI key
`export OPENAI_API_KEY=<YOUR_KEY>`
* Run the `rails` test
`pytest -v -m rails`


## Next up
- Docs! I'll raise another PR

## Checklist:

This checklist is a useful reminder of small things that can easily be forgotten.
Put an `x` in all the items that apply and remove any items that are not relevant to this PR.


[1]: http://chris.beams.io/posts/git-commit/
